_posts/zheinsohn/12-2-21-meetup-writeup-zheinsohn

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
So that your post displays on the site properly, could you change the date in the file title to `YYYY-MM-DD-` format and add a `.md` extension at the end to display as markdown?